### ISSUE TITLE
[Docker] Latest Tag Not Pulling The Latest Image

### ISSUE THREAD

**Issue by basudip** [abhinavsingh/proxy.py#1223]
When creating the container in ARM device using LATEST tag its not pulling latest stable release. In the time of writing the current stable version is V2.4.3. Docker Hub LATEST tagged image is almost 5 months old and don't have any ARM images.

**Comment by abhinavsingh** [comment]
Thanks for highlighting this.  Indeed, pushing multi-arch images to Docker was a left out task.  Can you try GCR which does contain multi-arch images https://github.com/abhinavsingh/proxy.py/pkgs/container/proxy.py

**Comment by basudip** [comment]
I think you understood wrong. Docker HUB version do have the multi-arch images. But LATEST Tagged image don't have it. you need to fix the latest tag only. other wise all development version have the multi-arch images. 

Latest GCR version points to v2.4.4rc2.dev6. Is that correct? Should not it be points to the last stable release?

**Comment by abhinavsingh** [comment]
@basudip Correct.  Issue is our release workflow currently don't push to Docker.   We need to get that fixed.  All releases you see on Docker were made manually by me.

On GCR, we publish every version, not just stable versions.  Hence you will see `.dev6` versions on GCR, as we push to GCR from every PR and when it gets merged into develop branch.

I'll look into how to get Docker releases automated too.
